.snippets/code/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[workspace.package]
+license = "MIT-0"
+authors = ["Parity Technologies <admin@parity.io>"]
```

Comment:
Probably has to be parity, papermoon and W3F. @albertov19?

Reply:
Agree

Reply:
Right, thanks will update to:

```rust
authors = ["Parity Technologies <admin@parity.io>", "Web3 Foundation", "PaperMoon.io"]
```